src/eval.rs

```diff
@@ -534,11 +533,7 @@ fn handle_non_by_name_attribute(
         // At this point, we completed two different checks for whether it's a `callPackage`.
         match (is_semantic_call_package, optional_syntactic_call_package) {
             // Something like `<attr> = { }`
-            (false, None)
-            // Something like `<attr> = pythonPackages.callPackage ...`
-            | (false, Some(_))
-            // Something like `<attr> = bar` where `bar = pkgs.callPackage ...`
-            | (true, None) => {
+            (_, None) | (false, Some(_)) => {
```

Comment:
Please revert or find another way of expressing the comments, like extracting to a function or something. Humans are bad at understanding what the `_` in these expressions actually _means_ in nixpkgs already, and the removal of the comments makes it much worse. I'd much rather have a longer function than to miss this.

Reply:
done: [5b4aae3](https://github.com/NixOS/nixpkgs-vet/pull/120/commits/5b4aae35ae10fc12bfed5888bba338bc34cc5fe3)